core/jazz_splunk-kinesis-log-streamer/components/utils.js

```diff
@@ -55,47 +122,189 @@ var getSubInfo = function (message, patternStr, index) {
   return result;
 }
 
-var getCommonData = function (payload) {
+var getCommonData = function (payload, config) {
+  // first get token for calling respective APIs
+  getToken(config)
```

Comment:
@SauarvD Since we expect tags only for sls logGroups, can we skip this chaining for non-sls logGroups?

Reply:
As discussed, we need the other api call to figure out whether the given logs are for sls-app or other, so we need the current structure for now.